ui/tx/TxSubHeading.tsx

```diff
@@ -3,45 +3,72 @@ import React from 'react';
 
 import config from 'configs/app';
 import useApiQuery from 'lib/api/useApiQuery';
+import { NOVES_TRANSLATE } from 'stubs/noves/novesTranslate';
 import { TX_INTERPRETATION } from 'stubs/txInterpretation';
 import AccountActionsMenu from 'ui/shared/AccountActionsMenu/AccountActionsMenu';
 import TxEntity from 'ui/shared/entities/tx/TxEntity';
 import NetworkExplorers from 'ui/shared/NetworkExplorers';
 import { TX_ACTIONS_BLOCK_ID } from 'ui/tx/details/txDetailsActions/TxDetailsActionsWrapper';
 import TxInterpretation from 'ui/tx/interpretation/TxInterpretation';
 
+import NovesSubHeadingInterpretation from './assetFlows/components/NovesSubHeadingInterpretation';
+
 type Props = {
   hash?: string;
   hasTag: boolean;
 }
 
+const feature = config.features.txInterpretation;
+
 const TxSubHeading = ({ hash, hasTag }: Props) => {
-  const hasInterpretationFeature = config.features.txInterpretation.isEnabled;
+  const hasInterpretationFeature = feature.isEnabled;
+  const isNovesInterpretation = hasInterpretationFeature && feature.provider === 'noves';
 
   const txInterpretationQuery = useApiQuery('tx_interpretation', {
     pathParams: { hash },
     queryOptions: {
-      enabled: Boolean(hash) && hasInterpretationFeature,
+      enabled: Boolean(hash) && (hasInterpretationFeature && !isNovesInterpretation),
       placeholderData: TX_INTERPRETATION,
     },
   });
 
-  const hasInterpretation = hasInterpretationFeature &&
+  const novesInterpretationQuery = useApiQuery('noves_transaction', {
+    pathParams: { hash },
+    queryOptions: {
+      enabled: Boolean(hash) && isNovesInterpretation,
+      placeholderData: NOVES_TRANSLATE,
+    },
+  });
+
+  const hasNovesInterpretation = isNovesInterpretation &&
+  (novesInterpretationQuery.isPlaceholderData || Boolean(novesInterpretationQuery.data?.classificationData.description));
+
+  const hasInternalInterpretation = (hasInterpretationFeature && !isNovesInterpretation) &&
     (txInterpretationQuery.isPlaceholderData || Boolean(txInterpretationQuery.data?.data.summaries.length));
 
+  const hasInterpretation = hasNovesInterpretation || hasInternalInterpretation;
+
   return (
     <Box display={{ base: 'block', lg: 'flex' }} alignItems="center" w="100%">
-      { hasInterpretation && (
-        <Flex mr={{ base: 0, lg: 6 }} flexWrap="wrap" alignItems="center">
-          <TxInterpretation
-            summary={ txInterpretationQuery.data?.data.summaries[0] }
-            isLoading={ txInterpretationQuery.isPlaceholderData }
-            fontSize="lg"
+      { hasInterpretation && hasNovesInterpretation ?
```

Comment:
I believe this change is not entirely accurate. If there is no interpretation, the `TxInterpretation` component will be displayed.